pkg/kube/util.go

```diff
@@ -15,8 +15,8 @@ import (
 )
 
 var genericPlaceholder, _ = regexp.Compile(`(?mU)<(.*)>`)
-var specificPathPlaceholder, _ = regexp.Compile(`(?mU)<path:(.+?)\#(.+?)>`)
-var indivPlaceholderSyntax, _ = regexp.Compile(`(?mU)path:(.+?)\#(.+?)`)
+var specificPathPlaceholder, _ = regexp.Compile(`(?mU)<path:(.+)\#(.+)>`)
+var indivPlaceholderSyntax, _ = regexp.Compile(`(?mU)path:(.+)\#(.+)`)
```

Comment:
I dont think this line should change. I would revert it back and leave `specificPathPlaceholder` as the only changed line

Reply:
it must not be a problem to leave it like it was but no need to be greedy here too. as you wish.